test/mitmproxy/addons/test_view.py

```diff
@@ -170,8 +171,10 @@ def test_reversed():
     assert v[0].request.timestamp_start == 3
     assert v[-1].request.timestamp_start == 1
     assert v[2].request.timestamp_start == 1
-    tutils.raises(IndexError, v.__getitem__, 5)
-    tutils.raises(IndexError, v.__getitem__, -5)
+    with pytest.raises(IndexError):
+        v.__getitem__(5)
```

Comment:
I didn't really look at the overall test code itself - just search&replace.
You can simplify it even further, the assignment is not required either.